modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/BaseCartResourceImpl.java

```diff
@@ -262,14 +254,22 @@ public Cart patchCart(
 			existingCart.setPurchaseOrderNumber(cart.getPurchaseOrderNumber());
 		}
 
-		if (cart.getShippingAddress() != null) {
-			existingCart.setShippingAddress(cart.getShippingAddress());
-		}
-
 		if (cart.getShippingAddressId() != null) {
 			existingCart.setShippingAddressId(cart.getShippingAddressId());
 		}
 
+		if (cart.getShippingMethod() != null) {
```

Comment:
Seems false positive because it has the same name as a component

Reply:
this is a string in the openAPI, should be added